### PR TITLE
Web Services Config for JDK 11

### PR DESCRIPTION
**Pull request by jonreding2010**
Copied JDK 8 config class and added to JDK 11
Copied JDK 8 config test class and added to JDK 11

### CODE REVIEW

**Comment by jonreding2010** [comment]
@bradyrad @jason-edstrom  I think the build failed because it's still compiling on JDK 8.

**Comment by jason-edstrom** [comment]
Nope.  Its running JDK 11.  The compiler config was missing the target tag in the pom.  It would make it through the tests and fail at the very end.

File: jmaqs-webservices-jdk11/pom.xml [hunk 1]
```diff
@@ -12,6 +12,14 @@
...
     <artifactId>jmaqs-webservices-jdk11</artifactId>
     <version>2.0.1-SNAPSHOT</version>
+    <dependencies>
+        <dependency>
+            <groupId>com.magenic.jmaqs.utilities</groupId>
+            <artifactId>jmaqs-utilities</artifactId>
+            <version>2.0.1-SNAPSHOT</version>
+            <scope>compile</scope>
+        </dependency>
+    </dependencies>
```
Comment (jason-edstrom) [inline jmaqs-webservices-jdk11/pom.xml:22]: Put this below the build block.  It looks really gross in the middle of the module definition information.

**Comment by jonreding2010** [comment]
Closing this PR because we can use JDK8 dependency to reduce duplication for now.

**Pull request closed without merging**